src/components/HorizontalScroller.module.scss

```diff
@@ -0,0 +1,109 @@
+@import "styles/modules/all";
+
+.horizontalScrollWrap {
+    position: relative;
+
+    .controls {
+        position: absolute;
+        top: 40%;
+        left: 20px;
+        transform: translateY(-50%);
+        z-index: 1;
+        width: calc(100% - 40px);
+        display: flex;
+        justify-content: space-between;
+        pointer-events: none;
+        z-index: 6;
+
+        @media (max-width: 760px) {
+            display: none;
+        }
+
+        button {
+            pointer-events: auto;
+            height: 40px;
+            width: 40px;
+            background-color: #004876;
+            border-radius: 50px;
+            z-index: 1;
+            border: 0 solid #858585;
+            color: #fff;
+            padding: 8px;
+            margin: 0;
+            cursor: pointer;
+
+            &:disabled {
+                opacity: 0.2;
+            }
+
+            img {
+                width: 100%;
+                height: 100%;
+                fill: #fff;
+                filter: invert(1);
+            }
+
+            &:nth-child(2) {
+                transform: rotate(180deg);
+            }
+        }
+    }
+
+    .viewPort {
+        .scrollWrap {
+            height: auto;
+            width: auto;
+            padding-bottom: 10px;
+            white-space: nowrap;
+            overflow-x: auto;
+            scroll-snap-type: x mandatory;
+            scroll-behavior: smooth;
+            padding-left: 5%;
+
+            .scrollItem {
+                max-width: 90%;
+                width: auto;
+                height: auto;
+                position: relative;
+                display: inline-flex;
+                flex-direction: column;
+                scroll-snap-align: center;
+                margin: 0 0px 0 0;
+                padding: 0 5px;
```

Comment:
What's the reason for this padding?

Reply:
for spacing, redundant with margin, removed, kept margin.